### ISSUE TITLE
UX issue. For small screens, the dropdown menu in the nav bar looks awkward. 

### ISSUE THREAD

**Issue by mugab686** [jburgos1/fls1lambdas#24]
![screen shot 2015-01-21 at 10 40 56 am](https://cloud.githubusercontent.com/assets/8159215/5839119/10fdf218-a15a-11e4-8230-34cc1628f0cd.png)

**Comment by mugab686** [comment]
The separation line makes the bar confusing.

**Comment by jburgos1** [comment]
Yeah, I noticed that too. Just not sure what the fix can be. I like the separation because it distinguishes the board vs the chapters. I think it's weird that the background color doesn't work for the separator. It might be something really easy to fix.
